lib/fog/compute/ecloud.rb

```diff
@@ -356,7 +356,14 @@ def set_extra_headers_for(params)
             "x-tmrk-version" => @version,
             "Date"           => Time.now.utc.strftime("%a, %d %b %Y %H:%M:%S GMT"),
           }.merge(params[:headers] || {})
-          if params[:method] == "POST" || params[:method] == "PUT"
+          if params[:method] == "POST" || params[:method] == "PUT" || params[:method] == "DELETE"
+            if !params[:headers]['Content-Length']
```

Comment:
This is a silly alert. Non-interpolated strings are more efficient for keys, not as good as symbols, but better than interpolated strings.